source/settings/plugin/quota-clone-plugin.rst

```diff
@@ -11,22 +11,26 @@ quota-clone plugin
 Settings
 ========
 
-.. dovecot_plugin:setting:: quota_clone_dict
+.. dovecot_plugin:setting:: quota_clone_dict_driver
```

Comment:
Lets try to do these consistently, see e.g. how mail_attachment does it. Although it's for fs, not dict, but it's still basically the same. Or if you think of some generic improvements to this, change all other @named_filters as well.